core/domain/visualization_registry_test.py

```diff
@@ -39,6 +40,34 @@ def test_get_visualization_class_with_invalid_id_raises_error(self):
             visualization_registry.Registry.get_visualization_class(
                 'invalid_visualization_id')
 
+    def test_visualization_class_with_invalid_option_names(self):
+        bar_chart = visualization_registry.Registry.get_visualization_class(
+            'BarChart')
+        bar_chart_instance = bar_chart('AnswerFrequencies', {}, True)
+
+        with self.assertRaisesRegexp(
+            Exception,
+            re.escape(
+                'For visualization BarChart, expected option names '
+                '[\'x_axis_label\', \'y_axis_label\']; received names []')):
+            self.assertEqual(bar_chart_instance.validate(), 12)
```

Comment:
I don't understand why object.validate() returns a value....

Reply:
That was a mistake. Fixed it!